components/http_foundation.rst

```diff
@@ -585,6 +585,65 @@ represented by a PHP callable instead of a string::
 
 .. _component-http-foundation-serving-files:
 
+Streaming a JSON Response
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 6.2
+
+The :class:`Symfony\\Component\\HttpFoundation\\StreamedJsonResponse` class was
+introduced in Symfony 6.2. It allows an API to return a lot of data as JSON and keep
+the used resources low by make usage of Generators.
+
+It expects an array which represents the JSON structure and the list which should be
+streamed are represented in the array as ``\Generator``. The response will stream this
```

Comment:
From what I understood of the implementation, any `Traversable` would do. Is there a reason to only mention generators?

Nice feature BTW :+1:

Reply:
Thx did add a hint that generators are recommended but traversable also works.